exercises/Basic-1-R.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: exercise
+title: Basic 1
+language: R
+---
+
+The number of birds banded at a series of sampling sites has been counted by
+your field crew and entered into the following list. The first item in each
+sublist is an alphanumeric code for the site and the second value is the number
+of birds banded. Cut and paste the list into your assignment and then answer the
+following questions by printing them to the screen.
```

Comment:
These aren't lists now, they are vectors, and there are two of them. The language should be updated to reflect this.